runtime/kreivo/src/payments.rs

```diff
@@ -28,29 +29,35 @@ impl pallet_payments::BenchmarkHelper<AccountId, FungibleAssetLocation, Balance>
 
 pub struct KreivoFeeHandler;
 
-const MANDATORY_FEE: bool = true;
 pub const SENDER_FEE: Percent = Percent::from_percent(1);
 pub const BENEFICIARY_FEE: Percent = Percent::from_percent(3);
 pub const INCENTIVE_PERCENTAGE: u8 = 10;
 
 impl FeeHandler<Runtime> for KreivoFeeHandler {
 	fn apply_fees(
 		asset: &AssetIdOf<Runtime>,
-		_sender: &AccountId,
+		sender: &AccountId,
 		_beneficiary: &AccountId,
 		amount: &Balance,
 		_remark: Option<&[u8]>,
 	) -> Fees<Runtime> {
 		let min = <Assets as fungibles::Inspect<AccountId>>::minimum_balance(*asset);
+
+		// Johan toca usar: try_from_sub_account
+		let is_paying_fees:bool = match PalletId::try_from_sub_account::<CommunityId>(sender) {
+			Some((pallet_id, _)) if pallet_id == crate::communities::CommunityPalletId::get() => true,
+			_ => false
```

Comment:
I guess is the contrary: communities don't pay, while others do.

```suggestion
			Some((pallet_id, _)) if pallet_id == crate::communities::CommunityPalletId::get() => false,
			_ => true
```

Reply:
This can be caught with proper tests